libbpf-tools/cachestat.c

```diff
@@ -164,8 +165,20 @@ int main(int argc, char **argv)
 	}
 	if (kprobe_exists("folio_account_dirtied")) {
 		bpf_program__set_autoload(obj->progs.kprobe_account_page_dirtied, false);
-	} else {
+		bpf_program__set_autoload(obj->progs.tracepoint__writeback_dirty_folio, false);
```

Comment:
Please separate kprobe and tracepoint checks.

Reply:
Hi, I'm not separate kprobe/tracepoint because those are same thing in diff kernel version.

* folio_account_dirtied:  v5.15
* account_page_dirtied: v2.6.29 - v5.15
* tp:writeback:writeback_dirty_page: v3.8 - v5.15
* tp:writeback:writeback_dirty_folio:  v5.15 - nowest

These observation points actually mean the same thing, so I put them together. Follow the if-else statement to choose one of them. Do you think it also makes some sense?